addon/templates/components/uni-footer-website-row-cell.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if header}}
+    <div class="uni-footer-website__row__cell__header">{{header}}</div>
```

Comment:
replace `{{header}}` with `{{{header}}}` as we can imagine that further on we may want to add more than just text here (like an icon/svg) this is just to be safe